src/kixi/event2s3/heartbeat_server.clj

```diff
@@ -0,0 +1,43 @@
+(ns kixi.event2s3.heartbeat-server
+  (require [clojure.java.io :as io]
+           [bidi.bidi :refer [tag]]
+           [bidi.vhosts :refer [make-handler vhosts-model]]
+           [com.stuartsierra.component :as component]
+           [taoensso.timbre :as timbre :refer [infof]]
+           [yada.yada :as yada :refer [resource]]
+           [yada.consume :refer [save-to-file]]
+           [yada.resources.webjar-resource :refer [new-webjar-resource]]))
+
+(defn bidi-routes []
+  ["" [["/health_check" (yada/handler "Peers are alive.\n")]]])
+
+(defn routes
+  "Create the URI route structure for our application."
+  [config]
+  [""
+   [(bidi-routes)
+    ;; This is a backstop. Always produce a 404 if we ge there. This
+    ;; ensures we never pass nil back to Aleph.
+    [true (yada/handler nil)]]])
+
+
+(defrecord WebServer [port listener]
+  component/Lifecycle
+  (start [component]
+    (if listener
+      component
+      (let [vhosts-model
+            (vhosts-model
+             [{:scheme :http :host (format "localhost:%d" port)}
```

Comment:
I think you'll need to make the vhost host configurable (e.g. https://github.com/MastodonC/kixi.datastore/blob/97cf0fb22a79539de2a81500391048b1f51aa0cf/resources/config.edn#L8)
We had some trouble getting Yada to work in mesos when this was set to localhost